proto/scripts/json_roundtrip_via_proto.py

```diff
@@ -33,7 +33,7 @@ def clean():
 proto_lhr = lhr_pb2.LighthouseResult()
 
 # fill proto lhr with data from JSON
-Parse(json.dumps(data), proto_lhr)
```

Comment:
I guess to finally figure out my real concern, is this something that anyone else will now have to change and match or are we the only people ever doing this proto-json conversion :)

apologies for being *very* slow on the uptake here 😆

Reply:
pretty sure the proto stuff has one user: us, with LR.

Reply:
awesome sauce